src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/Confirm.cshtml.cs

```diff
@@ -13,33 +14,61 @@ namespace SFA.DAS.ApprenticeCommitments.Web.Pages.Apprenticeships
     [Authorize]
     public class ConfirmApprenticeshipModel : PageModel
     {
-        private readonly AuthenticatedUserClient _client;
+        //private readonly AuthenticatedUserClient _client;
```

Comment:
Can we remove this comment or will it be needed later?

Reply:
The constructor was changed from taking AuthenticatedUserClient to IOuterApiClient & AuthenticatedUser as it now needs to make api calls - I discussed this with Ben and its commented as we may want AuthenticatedUserClient updated  to support api calls.  This can be reincluded/ removed once that decision is made